frontend/pages/Homepage/cards/MDM/_styles.scss

```diff
@@ -2,9 +2,6 @@
   margin-top: $pad-large;
   position: relative;
 
-  .data-table__wrapper {
-    overflow-x: auto;
```

Comment:
This caused funkiness with the tooltip staying inside the table and causing scrolling